src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -218,6 +219,60 @@ public void resetMaxHealth_MaxHealthChanged_ResetsBackTo20()
 		assertEquals(20.0, player.getHealth(), 0);
 	}
 
+	@Test
+	public void kill_PLayerDeathEventDispatched()
+	{
+		TestPlugin plugin = MockBukkit.load(TestPlugin.class);
+		AtomicBoolean eventDispatched = new AtomicBoolean();
+		server.getPluginManager().registerEvents(new Listener() {
+			@EventHandler
+			public void onDeath(PlayerDeathEvent event)
+			{
+				eventDispatched.set(true);
+			}
+		}, plugin);
```

Comment:
There is also `PluginManagerMock#assertEventFired(Predicate)` for this exact purpose.

Reply:
Ok, good to know. In this case, there are some other tests to refactor too.